### PR TITLE
Adds stolen Colonial Marines code to the server

### PR DESCRIPTION
**Pull request by Kierany9**
honk

- Replaces the Gooncode disk with a Colonial Marines Code disk.

### CODE REVIEW

**Comment by McBawbaggings** [comment]
>replaces goon disk

That alone is questionable.
If CM's staff want their code to be kept secret then it needs to self destruct on deconstruction.

**Comment by Pyko1** [comment]
Why remove the gooncode?

Could keep it and add this as a new joke instead.

**Comment by Badmex** [comment]
This should not remove the gooncode. This should just add it as another disk.
-1/3 for replacing the goon disk. You get a +1/3 if you add it as extra!

File: code/modules/research/research.dm [hunk 1]
```diff
@@ -323,3 +323,11 @@ research holder datum.
 	icon = 'icons/obj/items.dmi'
 	icon_state = "gooncode"
 	stored = new/datum/tech/programming{level = 4}
+	
+/obj/item/weapon/disk/tech_disk/gooncode
```
Comment (Carbonhell) [inline code/modules/research/research.dm]: change the path ending to cmcode

File: code/modules/research/research.dm [hunk 2]
```diff
@@ -323,3 +323,11 @@ research holder datum.
...
+	name = "Colonial Marines Code disk"
+	desc = "A disk containing Colonial Marines source code for further research."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "cmcode"
+	embed_chance = 85
```
Comment (Carbonhell) [inline code/modules/research/research.dm]: wut,why

**Comment by Pyko1** [comment]
I think 4 might be a bit too much even if its a pain in the ass to get.

**Comment by Kierany9** [comment]
[internal screaming]

**Comment by McBawbaggings** [comment]
[CM CODE WAS EXPUNGED AS BASED ON THE TERMS]

**Comment by GeneralGoose** [comment]
failure

**Pull request closed without merging**